Description: Upstream changes introduced in version 1.67.1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 pidgin-facebookchat (1.67.1-1) unstable; urgency=low
 .
   * New upstream release.
     - v1.67: + Plugin works with new 'Account Security'/'device
              login'/'register your computer' login security feature
     - v1.66: + Fix for crash on account creation
              + Fix for status messages showing as "like/unlike" rather
              than the status message
   * Switch to dpkg-source format 3.0 (quilt).
   * Remove quilt from control, rules.
   * Remove README.source.
   * Fix watch file by using googlecode.debian.net.  Closes: #581618.
 .
 The person named in the Author field signed this changelog entry.
Author: Iulian Udrea <iulian@ubuntu.com>
Bug-Debian: http://bugs.debian.org/581618

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- pidgin-facebookchat-1.67.1.orig/Makefile
+++ pidgin-facebookchat-1.67.1/Makefile
@@ -1,5 +1,5 @@
 #Customisable stuff here
-LINUX32_COMPILER = i686-pc-linux-gnu-gcc
+LINUX32_COMPILER = gcc
 LINUX64_COMPILER = x86_64-pc-linux-gnu-gcc
 WIN32_COMPILER = /usr/bin/i586-mingw32-gcc
 WIN32_WINDRES = i586-mingw32-windres
@@ -104,23 +104,13 @@ libfacebook-debug.dll:	${FACEBOOK_SOURCE
 libfacebookbsd60.so:	${FACEBOOK_SOURCES}
 	${FREEBSD60_COMPILER} ${LIBPURPLE_CFLAGS} -Wall ${GLIB_CFLAGS} -I. -g -O2 -pipe ${FACEBOOK_SOURCES} -o libfacebook.so -shared -fPIC -DPIC
 
-
-pidgin-facebookchat.exe:	libfacebook.dll
-	echo "Dont forget to update version number"
-	makensis facebook.nsi > /dev/null
 	
-pidgin-facebookchat.deb:	libfacebook.so libfacebookarm.so libfacebook64.so libfacebookppc.so
+pidgin-facebookchat.deb:	libfacebook.so
 	echo "Dont forget to update version number"
 	cp libfacebook.so ${DEB_PACKAGE_DIR}/usr/lib/purple-2/
-	cp libfacebookppc.so ${DEB_PACKAGE_DIR}/usr/lib/purple-2/
-	cp libfacebook64.so ${DEB_PACKAGE_DIR}/usr/lib64/purple-2/
-	cp libfacebookarm.so ${DEB_PACKAGE_DIR}/usr/lib/pidgin/
 	cp facebook16.png ${DEB_PACKAGE_DIR}/usr/share/pixmaps/pidgin/protocols/16/facebook.png
 	cp facebook22.png ${DEB_PACKAGE_DIR}/usr/share/pixmaps/pidgin/protocols/22/facebook.png
 	cp facebook48.png ${DEB_PACKAGE_DIR}/usr/share/pixmaps/pidgin/protocols/48/facebook.png
-	chown -R root:root ${DEB_PACKAGE_DIR}
-	chmod -R 755 ${DEB_PACKAGE_DIR}
-	dpkg-deb --build ${DEB_PACKAGE_DIR} $@ > /dev/null
 
 pidgin-facebookchat.tar.bz2:	pidgin-facebookchat.deb
 	tar --bzip2 --directory ${DEB_PACKAGE_DIR} -cf $@ usr/
